src/output.cpp

```diff
@@ -1965,6 +1965,23 @@ void insert_table( const catacurses::window &w, int pad, int line, int columns,
     wattroff( w, FG );
 }
 
+
+std::string display_satiety_bar( const int calpereffv )
```

Comment:
```suggestion
std::string satiety_bar( const int calpereffv )
```

The function does not actually display the bar, but only builds and returns the string, so it's probably better not to include the `display_` verb in the name.